### PR TITLE
 add doc string to AlignIO, Blast and condoalign module

### PR DESCRIPTION
**Pull request by michiboo**
This pull request addresses issue #1963

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Sure!!  You can put my name in the NEWS and CONTRIB file!

### CODE REVIEW

**Comment by peterjc** [comment]
It would have been better to start from the current master branch, but since your work on the Align module has just been merged, it doesn't actually cause any problems.

The automated style checks are failing due to missing full stops, and worse you've accidentally made a syntax error:

```
$ flake8 Bio/
Bio/codonalign/codonalignment.py:103:1: D400 First line should end with a period
Bio/codonalign/codonseq.py:267:48: E999 SyntaxError: invalid syntax
```

Have you installed ``flake8`` and the relevant plugins on your computer? Does this help:
https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst

File: Bio/codonalign/codonalignment.py [hunk 1]
```diff
@@ -100,6 +100,7 @@ def __getitem__(self, index, alphabet=None):
                                             generic_nucleotide)
 
     def get_aln_length(self):
+        """Get aligment length"""
```
Comment (peterjc) [inline Bio/codonalign/codonalignment.py]: Missing full stop.

File: Bio/codonalign/codonseq.py [hunk 2]
```diff
@@ -195,6 +195,7 @@ def translate(self, codon_table=default_codon_table,
         return "".join(amino_acids)
 
     def toSeq(self, alphabet=generic_dna):
+        """Convert DNA to sequence"""
```
Comment (peterjc) [inline Bio/codonalign/codonseq.py]: Missing full stop. I would probably be clearer to say Seq object?

File: Bio/codonalign/codonseq.py [hunk 3]
```diff
@@ -262,6 +264,7 @@ def ungap(self, gap=None):
                         rf_table=self.rf_table)
 
     @classmethod
+    """Get condon sequence from sequence data"""
```
Comment (peterjc) [inline Bio/codonalign/codonseq.py]: Missing full stop.

Comment (peterjc) [reply Bio/codonalign/codonseq.py]: Also this is in the wrong place (which is causing the syntax error), it should be below the ``def`` line, not above it.

**Comment by michiboo** [comment]
Hi, fixed as required, sorry for the small mistake, I am new to open source!

File: Bio/codonalign/codonseq.py [hunk 4]
```diff
@@ -263,6 +265,7 @@ def ungap(self, gap=None):
 
     @classmethod
     def from_seq(cls, seq, alphabet=default_codon_alphabet, rf_table=None):
+        """Get condon sequence from sequence data."""
```
Comment (peterjc) [inline Bio/codonalign/codonseq.py]: Typo, condon -> codon.

**Comment by peterjc** [comment]
This is looking good - just one tiny typo to fix (which I should have spotted last time), and we can apply your changes.

Do you want to add your name to the NEWS and CONTRIB file?

**Comment by peterjc** [comment]
Branched rebased and updated to add name... assuming the tests still pass, this looks ready to merge.

**Comment by peterjc** [comment]
Merged, thank you!

**Pull request merged**